src/mobile/src/main/java/tupperdate/android/ui/theme/material/BrandedButton.kt

```diff
@@ -36,10 +36,10 @@ import tupperdate.android.ui.theme.Smurf800
 import java.util.*
 
 
-private val FirstColor = ColorPropKey()
-private val SecondColor = ColorPropKey()
-private val ThirdColor = ColorPropKey()
-private val FourthColor = ColorPropKey()
+private val FirstColor = ColorPropKey(label = "A")
+private val SecondColor = ColorPropKey(label = "B")
+private val ThirdColor = ColorPropKey(label = "C")
+private val FourthColor = ColorPropKey(label = "D")
```

Comment:
Ew... Why did they do that...

Reply:
There's a new warning if PropKeys are not named, because of the Compose animations inspector in AS. We have to label properties so they can somehow be displayed in the IDE.

I don't like it either, but I'd rather have constants than warnings 😄